samples/hono/src/handlers/createPets.ts

```diff
@@ -1,12 +1,14 @@
 import { createFactory } from 'hono/factory';
 import { zValidator } from '@hono/zod-validator';
-import { CreatePetsContext, createPetsBody } from '../petstore';
+import { CreatePetsContext } from '../petstore.context';
+import { createPetsBody } from '../petstore.zod';
 
 const factory = createFactory();
 
+
 export const createPetsHandlers = factory.createHandlers(
-  zValidator('json', createPetsBody),
-  (c: CreatePetsContext) => {
-    return c.json({ message: 'createPets handler' });
+zValidator('json', createPetsBody),
+(c: CreatePetsContext) => {
```

Comment:
Since there is no longer a default implementation, `c` is now unused. Depending on the user's formatter settings, a warning will occur.
Is this an intended change?